src/AspNet.Security.OAuth.Kloudless/KloudlessAuthenticationHandler.cs

```diff
@@ -54,7 +55,19 @@ public class KloudlessAuthenticationHandler : OAuthHandler<KloudlessAuthenticati
 
             var principal = new ClaimsPrincipal(identity);
             var context = new OAuthCreatingTicketContext(principal, properties, Context, Scheme, Options, Backchannel, tokens, payload.RootElement);
-            var accounts = payload.RootElement.GetProperty("objects");
+
+            var isObjectsExists = payload.RootElement.TryGetProperty("objects", out var accounts);
+            if (!isObjectsExists)
+            {
+                throw new ArgumentException("Property objects is missing on accounts JSON");
```

Comment:
This isn't really an argument exception. If there isn't a better candidate, suggest using `InvalidOperationException`.